### ISSUE TITLE
Update on-site training schedule in documentation

### ISSUE THREAD

**Issue by redshiftzero** [freedomofpress/securedrop-docs#64]
There is a great outline in the [SecureDrop On-Site Training Schedule](https://docs.securedrop.org/en/latest/training_schedule.html) for getting new admins and journalists up to speed on SecureDrop. 

This would be even stronger if it the outline were fleshed out and each section linked to a piece of documentation describing how to go about doing each task (for many of the tasks listed the documentation currently exists, if it doesn't, we should create it).

**Comment by huertanix** [comment]
Wondering whether it'd be worth updating for remote trainings, since trainings have been split across longer than a few days, but also it's hard to say how long we'll be needing to do these remotely.

**Comment by conorsch** [comment]
Certainly, @huertanix, that'd be a great update. Ideally, the URL for the training schedule would provide a concise overview to external folks, and always provide an up-to-date summary of our expectations of how a given training will run. Given how much is remote these days, that should definitely be spoken to in the docs.

**Comment by huertanix** [comment]
^ Aforementioned branch updates the schedule to match what DST is doing now. Not every bullet point links to a guide about a particular tasks, but the to-do list for the period right after an install is generally not inclusive of things that may/may not happen in 1+ years (rotating application private keys, etc). I figure those should be left to be answered in support questions, which admins will definitely have since they're not going to retain something they ran through in the midst of a whirlwind of installs and training a year before they have to do it. Also, this is to keep the training sessions within their time box. Going through 27 topics for admin training doesn't really fit in a 2-hour period.

**Comment by nathandyer** [comment]
In our proposal session today, we decided that it's best to remove this from the public-facing SecureDrop docs. It's worth keeping some form of this around that we can share with organizations that are interested in a support contract, or perhaps it might make sense on the securedrop.org website, but not as part of the main project documentation.
